code/datums/outfits/ert/iac.dm

```diff
@@ -22,7 +22,9 @@
 		/obj/item/storage/firstaid/surgery = 1,
 		/obj/item/storage/box/gloves = 1,
 		/obj/item/storage/box/syringes = 1,
-		/obj/item/device/flashlight/pen = 1
+		/obj/item/device/flashlight/pen = 1,
+		/obj/item/gun/energy/pistol = 1,
+		/obj/item/clothing/accessory/holster/armpit = 1
```

Comment:
Ditto. They cant use the holster since they already get a webbing.

Reply:
It gives them the option to take the vest off.

Reply:
But why would you ever do that? It's less storage space. It makes no sense to do it.

Reply:
Just an option for them to have if they are going into a place where they know for sure they'll be shot at. Sometimes space is a needed sacrifice for safety.